olsync/olclient.py

```diff
@@ -33,6 +33,15 @@ class OverleafClient(object):
     uploading a file to a project.
     """
 
+    # FIXED: use this method to filter project, not archived and not trashed
```

Comment:
I'd remove this comment since I rather look at the git history to check what has been done